client/app/dedicatedUniverseComponents/price/price.filter.js

```diff
@@ -55,27 +55,29 @@ export default /* @ngInject */ ($translate) => {
     US: usTouch,
   };
 
-  function format(price, paramCountry) {
+  function format(price, paramCountry, isMonthly) {
     const country = (angular.isString(paramCountry) && paramCountry) || 'FR';
     const taxes = showTaxes[country];
+    const isMonthlySuffix = isMonthly ? '_isMonthly' : '';
```

Comment:
```suggestion
    const isMonthlySuffix = isMonthly ? '_monthly' : '';
```

In that way we can keep the translation key in lowerCase/snakeCase.